### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by ieaves**

### CODE REVIEW

File: examples/deployment/docker-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+
+```bash
+docker volume create --driver local \
```
Comment (edlouth) [inline examples/deployment/docker-compose/README.md:4]: I think this readme needs more or removing, it doesn't make a whole load of sense at the moment.

File: examples/deployment/docker-compose/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,111 @@
+version: "3.7"
```
Comment (edlouth) [inline examples/deployment/docker-compose/docker-compose.yml:1]: Where is this file referenced from? I can't see a link in the readme

File: README.md [hunk 3]
```diff
@@ -22,29 +22,66 @@
...
+You can find a full quickstart guide in the [documentation](https://docs.grai.io/quick-start) which covers deploying your own instance of Grai and getting set up with your first connector in python.
+
+
+### Docker
+
+You can always find pre-built images of the backend server at `ghcr.io/grai-io/grai-core/grai-server:latest` and the frontend at `ghcr.io/grai-io/grai-core/grai-frontend:latest`, however, you prefer to build from source you can do so with docker compose.
 
 ```
 git clone https://github.com/grai-io/grai-core
 cd grai-core/grai-server
```
Comment (edlouth) [inline README.md:75]: Should we reference the example here? we could copy the example to top level and run from there, then have that ignored by source control and lets the user customise as they see fit.
```suggestion
cp examples/deployment/docker-compose/docker-compose.yml .
```

File: examples/deployment/docker-compose/README.md (hunk 1 above)
Comment (ieaves) [reply examples/deployment/docker-compose/README.md:4]: Yeah that was from a previous iteration where I was fucking with celery.

**Review by edlouth — COMMENTED** [review]
Not sure we need this file `examples/deployment/k8s/helm/.gitkeep`

**Review by edlouth — APPROVED** [review]

**Pull request merged**